crates/wasmcloud-host/src/host.rs

```diff
@@ -224,7 +224,7 @@ impl Host {
         link_name: Option<String>,
     ) -> Result<()> {
         let hc = HostController::from_hostlocal_registry(&self.id.borrow());
-        let bytes = fetch_oci_bytes(cap_ref, self.allow_latest).await?;
+        let bytes = fetch_oci_bytes(cap_ref, self.allow_latest, false).await?;
```

Comment:
Similar to above, the `allow_insecure` could be included on the Host struct (you will also want to include a method in the HostBuilder struct to allow insecure connections. If you put the option on the Host, then each host can configure if they allow insecure registries.